jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -275,3 +286,180 @@ def data_outside_gwcs_bounding_box(data, x, y):
         if not (bb_xmin <= x <= bb_xmax and bb_ymin <= y <= bb_ymax):
             outside_bounding_box = True  # Has to be Python bool, not Numpy bool_
     return outside_bounding_box
+
+
+def _get_fits_wcs_from_file(filename):
+    header = fits.getheader(filename)

```

Comment:
Without providing extension, this is going to be wrong for MEF with multiple SCI extensions. What if I want `('SCI', 2)`?